.github/workflows/claim-crates.yml

```diff
@@ -0,0 +1,28 @@
+name: Claim Crates
+
+on:
+  push:
+    branches:
+      - master
+
+jobs:
+  claim-crates:
+    strategy:
+      matrix:
+        os: ["ubuntu-latest"]
```

Comment:
why use a matrix for this?

Reply:
Other workflows do so just staying consistent.

Reply:
hmm i don't think it makes sense here since this will never need to be ran on more than one

Reply:
Sure I'll remove it then. And from the other publish CI